benchmark/alpa/benchmark.py

```diff
@@ -88,16 +88,16 @@ def benchmark_suite(suite_name,
         (parameter_count, peak_mem, latencies, tflops, metadata) = result
 
         heads = [
-            "Type", "Model Config", "#Microbatch", "#GPU", "Parallel Config",
-            "Mean Time (s)", "Std Time (s)", "#Params (Billion)", "TFLOPs",
-            "Peak Mem (GB)", "Metadata"
+            "ModelName", "BS", "#Microbatch", "DP", "OP", "PP", "#GPU",
+            "MeanTime(s)", "StdTime(s)", "TFLOPs", "Weights(GB)", "PeakMem(GB)",
+            "StageLatencies(s)"
```

Comment:
This script is shared by a lot of other benchmarks. Not all benchmarks have these fields. The default value `n_iters=10` is not good for other benchmarks.

We can either
1. Copy and port related things to mms repo or mms branch, so we don't need to worry about these compatibility issues in the future.
2. or add log in benchmark_one_case_gpt_bert_inference